Directory.Build.props

```diff
@@ -382,6 +381,10 @@
     <DocumentationFile Condition="'$(DocumentationFile)' == ''">$(OutputPath)$(MSBuildProjectName).xml</DocumentationFile>
   </PropertyGroup>
 
+  <PropertyGroup Condition="'$(IsSourceProject)' == 'true' OR '$(IsReferenceAssembly)' == 'true'">
```

Comment:
So the shims are neither source projects, ref assemblies, nor test projects?

Reply:
Correct. Based on our regex they are not.

Reply:
src: https://github.com/dotnet/corefx/blob/8fd630e7d1573af7043c19818572be667a9ba877/Directory.Build.props#L21
ref: https://github.com/dotnet/corefx/blob/901a1d554a7bcb489eb363fb724c9d294ec35940/eng/ReferenceAssemblies.props#L3